portal-impl/src/com/liferay/portal/struts/JSONAction.java

```diff
@@ -77,7 +78,15 @@ public ActionForward execute(
 			json = getJSON(actionMapping, actionForm, request, response);
 
 			if (Validator.isNotNull(callback)) {
-				json = callback + "(" + json + ");";
+				StringBundler sb = new StringBundler(5);
+
+				sb.append("/**/");
```

Comment:
Recommend method to fix is to prepend the callback with `/**/`
See https://miki.it/blog/2014/7/8/abusing-jsonp-with-rosetta-flash/#mitigations-by-website-owners